de

```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env bash
-
-# You should set the following environment variables:
-#   $DE_START_NETWORK_PSK: Password/PSK of your current network
-#   $DE_ROBOT_NETWORK_PSK: ...of your robot network
-#   $DE_ROBOT_NETWORK_SSID: The SSID of your robot's network
+source ~/.de
```

Comment:
This is how I was hoping you would do it. Perhaps it might be a good idea to check if the file exists, and print an error if it does not? https://stackoverflow.com/questions/638975/how-do-i-tell-if-a-regular-file-does-not-exist-in-bash